cou/logging.py

```diff
@@ -76,7 +78,10 @@ def setup_logging(log_level: str = "INFO") -> None:
 
     root_logger.addHandler(log_file_handler)
     root_logger.addHandler(console_handler)
-    logger.info("Logs of this execution can be found at %s", file_name)
+    log_file_msg = f"Logs of this execution can be found at {file_name}"
+    if not quiet:
+        print(log_file_msg)
+    logger.info(log_file_msg)
```

Comment:
What is purpose to log this message to log file? and If I do `-v` I will see print and info log message at once. I think we should remove the logging message.